classesExamples/firstFile.txt

```diff
@@ -0,0 +1 @@
+This is nice 2
```

Comment:
What will happen next time you execute the code, will this be overwritten? Will it happen every time then? Can you check that the content is already that one, and not write it again?